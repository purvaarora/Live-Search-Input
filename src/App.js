import logo from './logo.svg';
import './App.css';
import LiveSearch from './components/LiveSearch/LiveSearch';

function App() {
  return (
    <div className="App">
      <LiveSearch/>
    </div>
  );
}

export default App;
