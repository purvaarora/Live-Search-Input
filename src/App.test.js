import React from 'react';
import renderer from 'react-test-renderer'
import App from './App';

describe('App component tests', () => {
  it('matches snapshot', () => {
    const component = renderer.create(
      <App />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
});
