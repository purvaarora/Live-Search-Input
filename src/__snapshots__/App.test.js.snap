// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component tests matches snapshot 1`] = `
<div
  className="App"
>
  <div
    className="liveSearch"
  >
    <input
      className="liveSearch-input"
      id="liveSearchInput"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onMouseDown={[Function]}
      placeholder="Choose Manager"
      type="text"
      value=""
    />
  </div>
</div>
`;
